### DISCUSSION TITLE
Scrollbar handle has become nearly totally black in `v0.3.35`

### DISCUSSION THREAD

**Discussion by rotemdan** [open-webui/open-webui#6472] [General]
Now almost invisible in the dark mode:

![Screenshot_1](https://github.com/user-attachments/assets/cbb3d340-58cd-4992-97cd-3aac1b9bd4a1)

**Comment by silentoplayz** [comment]
I can attest to this. There is a styling difference between the scrollbars within a chat on Firefox compared to Chrome.

**Comment by rotemdan** [comment]
Yes, that screenshot is on Brave browser (Chromium, Windows 11)

On latest Firefox it looks like this:

![Screenshot_1](https://github.com/user-attachments/assets/4f9b5d76-466a-4be0-8b80-3a3b42ac1b61)

(still a bit too narrow for my preference [unless hovered], but visible)

**Comment by WesleyBatista** [comment]
On a Mac, Google Chrome, not only the dark mode but light mode too. It's super hard to find the scrollbar. Like @rotemdan, I also find it to narrow 😣 :

<kbd><img width="408" alt="Screenshot 2024-12-14 at 01 13 05" src="https://github.com/user-attachments/assets/6c63dbfe-789a-4769-8d40-3aa71f56f71c" /></kbd>

Considering both UX and accessibility, can we rely on browser standards for scrollbars, or maybe get inspired by Github's approach to it (very accessible imo)?
